src/manage/synchronisation.md

```diff
@@ -1,16 +1,36 @@
 # Synchronisation and Conflicts
```

Comment:
thinking of the name... do you want to include also here the other sync concepts, something that is on the README in geodiff? like also explain the "good" situation what geodiiff is able to do ?